helm/core/templates/controller-service.yaml

```diff
@@ -23,6 +23,12 @@ spec:
     - port: 15014
       name: http-monitoring # prometheus stats
       protocol: TCP
+    {{- if .Values.apiserver.enabled }}
+    - port: {{ .Values.apiserver.securePort }}
+      name: https-apiserver # mTLS with k8s-signed cert
```

Comment:
I don't think it has anything to do with k8s-signed cert.

Reply:
It is generated by copilot, sorry.